packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -38,14 +38,15 @@ export class CheckpointImpl implements Checkpoint {
   timestamp: number;
   metadata: Map<string, string>;
   asa: Map<string, ASAInfo>;
-  ssc: Map<string, SSCInfo>;
+  ssc: Map<string, Map<number, SSCInfo>>;
   dLsig: Map<string, LsigInfo>;
 
   constructor (metadata?: Map<string, string>) {
     this.timestamp = +new Date();
     this.metadata = (metadata === undefined ? new Map<string, string>() : metadata);
     this.asa = new Map<string, ASAInfo>();
-    this.ssc = new Map<string, SSCInfo>();
+    const mp = new Map<number, SSCInfo>();
```

Comment:
Maybe we can add a type `type TimeStamp = number`, and use that type here for more clarity?
```js
const mp = new Map<TimeStamp, SSCInfo>();
```

Reply:
yes, good idea